### DISCUSSION TITLE
css stylesheet

### DISCUSSION THREAD

**Discussion by ignacio-gomez** [bootstrap-email/bootstrap-email#163] [Help] [answered]
What's the right way to include a stylesheet. I'd like to customize font-family, background, types etc.
Thank you

**Comment by stuyam** [comment, marked as answer]
Check out the *Customize Sass* section of the docs for info about how to do that: https://bootstrapemail.com/docs/configure#customize-sass Let me know if you have any issues.

**Reply by krzcho** [reply]
Similar question: I would like to include simple "click checkbox to see details" trick with the following style:
```
pre {
  display: none
}

#hidden-checkbox:checked+pre {
  display: block
}

Click to show more <input id="hidden-checkbox" type="checkbox"><pre>details</pre>
```
Doable with bootstrap email? Styling must be left intact in the head

I have placed it successfully bootstrap-head.scss below /*! allow_purge_after */ and it works fine in email preview on dev site; however production site applies PRE style inline - how to prevent it?
